src/dispatch/batch.rs

```diff
@@ -3,6 +3,7 @@ use crate::{
     System, SystemData, World,
 };
 
+// XXX Make it non-pub
```

Comment:
ah, missed this removal

Reply:
I think it can stay public (there's not much reason why not, though the usefulness for the „ordinary“ user is probably questionable). So I've removed the comment.